src/file_uploader/pinata.rs

```diff
@@ -88,6 +88,11 @@ pub fn ipfs_hash_to_tgz_url(hash: &str) -> String {
     format!("{pinata_domain}/ipfs/{hash}?filename={TARBALL_NAME}")
 }
 
+pub fn ipfs_hash_to_docs_url(hash: &str) -> String {
+    let pinata_domain = env::var("PINATA_URL").expect("PINATA_URL must be set");
+    format!("{pinata_domain}/ipfs/{hash}?filename=docs.tgz")
```

Comment:
nit: `docs.tgz` magic string - might be worthwhile declaring `DOCS_TARBALL_NAME: &str = "docs.tgz"`